packages/foreman/foreman/foreman.spec

```diff
@@ -672,6 +672,7 @@ Group:  Applications/System
 Requires: %{?scl_prefix}rubygem(puma) >= 5.1
 Requires: %{?scl_prefix}rubygem(puma) < 6.0
 # end specfile service Requires
+Requires: %{?scl_prefix}rubygem(puma-status)
```

Comment:
Should `puma-status` be added to `bundler.d/service.rb` in core?

We'll need to smuggle it in there for Debian packaging anyways, and it feels cleaner to just add it in core instead of packaging?

Reply:
According to https://github.com/theforeman/foreman/pull/7578#discussion_r405930554 it shall not, and I am not going up any hills today.